apps/test/unit/templates/instructions/HeightResizerTest.jsx

```diff
@@ -6,18 +6,24 @@ import HeightResizer from '@cdo/apps/templates/instructions/HeightResizer';
 
```

Comment:
(not required, but would be nice) Since you're here, mind replacing deprecatedChai with reconfiguredChai?